static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -122,6 +133,49 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
   );
 }
 
+function TraceDataSection({
+  eventData,
+  crashReportId,
+  hasProject,
+}: {
+  crashReportId: string | undefined;
+  eventData: Event;
+  hasProject: boolean;
+}) {
+  // If there's a linked error from a crash report and only one other issue, showing both could be redundant.
+  // TODO: we could add a jest test .spec for this ^
+  const {oneOtherIssueEvent, traceEvents, isLoading, isError} = useTraceTimelineEvents({
+    event: eventData,
+  });
+  const hide: boolean =
```

Comment:
we don't usually annotate variables like this, it'll be a boolean, or truthy, it's fine.

```suggestion
  const hide =
```

Reply:
It might be a good idea to rephrase this as `const show = ...` and flip the logic around. That way the variable is more proactive sounding and the return ternary would be a little easier as `return show && ffEnabled ? <Section>...</Section> : null;`

phrasing conditions like `if (yes && yes) ...` is easier to understand than `if (no || no) ... else ...` if that makes any sense.